apps/ryal_core/lib/ryal/web/models/payment_gateways/stripe.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Ryal.PaymentGateway.Stripe do
+  @moduledoc "Relevant functions for working with the Stipe API."
+
+  alias Ryal.PaymentGatewayQuery
+
+  @stripe_api_key Map.get(Ryal.payment_gateways(), :stripe)
+  @stripe_base "https://#{@stripe_api_key}:@api.stripe.com"
+
+  @spec create(atom, Ecto.Schema.t, String.t) :: {:ok, String.t}
```

Comment:
At this level I agree.  We shouldn't tie gateway usage to Ecto dependencies....Unless we are comitted to building on the Phoenix stack instead of a more general approach.